### PR TITLE
create Email reader extension 

### PR DESCRIPTION
**Pull request by az4521**
Zawinski's law states that "Every program attempts to expand until it can read mail. Those programs which cannot so expand are replaced by ones which can"

in order to ensure that tachiyomi is not replaced by another program, i have created a simple Email reader extension.

### CODE REVIEW

File: src/all/email/release/output-metadata.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (jobobby04) [inline src/all/email/release/output-metadata.json:1]: Forgot to remove this file and the apk :harold:

**Comment by AbdullahM0hamed** [comment]
This is an amazing, life changing extension. With this extension, I can now read my emails alongside my manga without ever having to leave the Tachiyomi app!

I kindly request that the maintainers merge this awesome extension at their earliest convenience.

Thanks in advance

**Comment by az4521** [comment]
croppedImage_1609565873300
Screenshot_20210101-231645_TachiyomiAZ
Screenshot_20210102-020339_TachiyomiAZ

here are some screenshots of the extension

**Review by AbdullahM0hamed — APPROVED** [review]

**Comment by as280093** [comment]
If email can be converted into images can't novel's too

**Comment by az4521** [comment]
yes but text based extensions normally get rejected. this extension is a meme

**Comment by AbdullahM0hamed** [comment]
@as280093 https://github.com/tachiyomiorg/tachiyomi-extensions/pull/3506 - will be rejected

**Review by CrepeTF — APPROVED** [review]
Before this extension, my life was miserable. I kept missing my emails because of my very real chronic pornhwa and hentai reading disorder, but now, I can read porn, AND switch to my emails in just 6 clicks! Highly recommended.

**Comment by BraveDude8** [comment]
Thank god I don't need to keep an icky email app installed on my dedicated Tachiyomi 32" color E-Reader anymore. Keep up the good work.

**Comment by arkon** [comment]
This is genius and all, but I don't want to deal with people (re: the average user) asking what the heck this is and how it works and what the security implications are.

Feel free to include it in AZ or something though, it should definitely live on in some form.

**Pull request closed without merging**